warp-core/src/main/scala/com/workday/warp/persistence/CoreQueries.scala

```diff
@@ -391,53 +404,127 @@ trait CoreQueries extends AbstractQueries {
 
 
   /**
-   * Creates a [[DBIO]] for inserting or updating `row` into [[TestExecutionTag]] and returning an [[Option]] with
-   * the result
-   *
+   * Creates a [[DBIO]] for inserting or updating `row` into [[TestExecutionTag]] and returning a [[DBIO]] with the
+   * row created. This updates the `value` field.
    * @param row to be inserted
-   * @return a [[DBIO]] (not yet executed) for inserting or updating `row` into [[TestExecutionTag]]
+   * @tparam T TestExecutionTagRowLikeType
+   * @return DBIO of TestExecutionTagRow updated
    */
-  override def insertOrUpdateTestExecutionTagQuery[T: TestExecutionTagRowLikeType](row: T): DBIO[Option[TestExecutionTagRowWrapper]] = {
-    TestExecutionTag returning TestExecutionTag.map(_.idTestExecutionTag) into (
-      (row, id) => row.copy(idTestExecutionTag = id)
-    ) insertOrUpdate(row)
+  override def insertOrUpdateTestExecutionTagValueQuery[T: TestExecutionTagRowLikeType](row: T): DBIO[TestExecutionTagRowWrapper] = {
+    def byId = TestExecutionTag.filter(t =>
```

Comment:
lets rename these subqueries since they no longer looking up by id like in an earlier iteration. 

for the non-metatag versions, we can also make them vals instead of defs. 

```scala
    val find: Query[Tables.TestExecutionTag, Tables.TestExecutionTagRow, Seq] = TestExecutionTag.filter(t =>
      t.idTestExecution === row.idTestExecution && t.idTagName === row.idTagName
    )

    for {
      found: Option[TestExecutionTagRow] <- find.result.headOption
      _ <- found match {
        case Some(existingTag: TestExecutionTagRow) =>
          TestExecutionTag.filter(_.idTestExecutionTag === existingTag.idTestExecutionTag)
          .map(_.value)
          .update(row.value)
        case None =>
          this.writeTestExecutionTagQuery(row)
      }
      tag: TestExecutionTagRow <- find.result.head
    } yield tag
```

feel free to call it something other than `find` (like `findQuery`, perhaps) if you like, and feel free to omit the `Query[Tables.TestExecutionTag, Tables.TestExecutionTagRow, Seq]` type annotation